### ISSUE TITLE
a11y: Color Contrast - Preferences

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13676]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio: 3.16:1 
Main Content: Back

Foreground color: #FFFFFF
Background color: #B6B6C2
Contrast ratio: 2.00:1 
Main Content: 
- [ ] English, Mon, Nov 4 2024, Mon, 11 Nov 2024, 100, 50, local and so on (dropdown option in selected state when keyboard focus is not on option in ""Language"", ""Date Format"", ""Time Format"" and so on drop-downs)

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.16:1 
Main Content:
- [ ] English, Mon, Nov 4 2024, Mon, 11 Nov 2024, 100, 50, local and so on (dropdown option in selected state when keyboard focus is on option and on hover in ""Language"", ""Date Format"", ""Time Format"" and so on drop-downs)
- [ ] Light, Auto, Emacs, Show Releases Only and so on in selected state 

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

## Recommendation

On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.

**Comment by rak-phillip** [comment]
Fails the VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
> 
> Instances :
> Foreground color: #6C6F76
> Background color: #EEF0F4
> Contrast ratio: 4.4:1
> Main Content:
> - Light, Auto, Emacs, Show Releases Only and so on in hover state 
>
> Foreground color: #FFFFFF
> Background color: #B6B6C2
> Contrast ratio: 2.00:1 
> Main Content: 
> - English, Mon, Nov 4 2024, Mon, 11 Nov 2024, 100, 50, local and so on (dropdown option in selected state when keyboard focus is not on option in ""Language"", ""Date Format"", ""Time Format"" and so on drop-downs)
>
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 78
